zerver/worker/queue_processors.py

```diff
@@ -342,6 +345,56 @@ def consume(self, event):
         mirror_email(email.message_from_string(event["message"].encode("utf-8")),
                      rcpt_to=event["rcpt_to"], pre_checked=True)
 
+@assign_queue('outhook_worker')
```

Comment:
This worker addition should come with changes to `./puppet/zulip/files/supervisor/conf.d/zulip.conf` to add the queue workers there as well.  See http://zulip.readthedocs.io/en/latest/queuing.html for details.